testing/pastel/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname=pastel
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="A command-line tool to generate, analyze, convert and manipulate colors"
+url="https://github.com/sharkdp/pastel"
+arch="x86_64"
```

Comment:
How come this doesn't work on other arches?

Reply:
builds fine on x86_64 and aarch64 but fails a single test on x86 armhf armv7.
upstream report: https://github.com/sharkdp/pastel/issues/97